### PR TITLE
Add 2 bots and alphabetize the lists

### PR DESCRIPTION
**Pull request by rojvv**

### CODE REVIEW

**Comment by lodthe** [comment]
You should add new awesome items to the end of lists. When you sort them in alphabetical order, it's harder to find new items.
If you need to find something particular, you can use the search function (ctrl + f) in your browser.

**Comment by rojvv** [comment]
@lodthe I’m resetting this...

File: README.md [hunk 1]
```diff
@@ -29,3 +29,6 @@ Feel free to improve this list by [contributing](CONTRIBUTING.md).
 
 - [TGVCBOT](https://github.com/thehamkercat/Telegram_VC_Bot) - A bot that can play music from various sources in VC.
 - [callsmusic/tgvc-userbot](https://github.com/callsmusic/tgvc-userbot) - A Voice Chat UserBot with playlist and Heroku support.
+- [callsmusic/callsmusic-1](https://github.com/callsmusic/callsmusic-1) - A bot based on [MarshalX/tgcalls](https://github.com/MarshalX/tgcalls) with queue & multiple chats support.
+- [callsmusic/callsmusic-2](https://github.com/callsmusic/callsmusic-2) - Same as above, but based on [pytgcalls/pytgcalls](https://github.com/pytgcalls/pytgcalls).
```
Comment (MarshalX) [inline README.md]: "Same as above" I think that each point of list should be isolated from another

Comment (rojvv) [reply README.md]: OK, I’m isolating them...

Comment (rojvv) [reply README.md]: Done, check now!

**Comment by MarshalX** [comment]
Thanks!

**Pull request merged**